lib/wise_homex/models/expense.ex

```diff
@@ -7,15 +7,17 @@ defmodule WiseHomex.Expense do
 
   embedded_schema do
     belongs_to(:fiscal_year, WiseHomex.FiscalYear, type: :binary_id)
```

Comment:
Minor nit: Perhaps also remove parens from this line for consistency?

Reply:
Of course, I missed that.... Done in https://github.com/wise-home/wise_homex/pull/193/commits/21e93ede3c610cf15d3d87c22148a26026fb1716